module/validation/approval_validator_test.go

```diff
@@ -0,0 +1,118 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/flow-go/engine"
+	"github.com/onflow/flow-go/module"
+	mock2 "github.com/onflow/flow-go/module/mock"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+func TestApprovalValidator(t *testing.T) {
+	suite.Run(t, new(ApprovalValidationSuite))
```

Comment:
could we please add a test that approvals from ejected nodes are considered invalid: https://github.com/onflow/flow-go/blob/ce9931b097495760658d4693eed31ae3066a1950/module/validation/common.go#L49-L52